src/Magnum/GL/Test/BufferGLTest.cpp

```diff
@@ -254,10 +255,10 @@ void BufferGLTest::bindRange() {
 
     MAGNUM_VERIFY_NO_GL_ERROR();
 
-    /** @todo C++14: get rid of std::make_tuple */
+    using BufferOffset = Containers::Triple<Buffer*, GLintptr, GLsizeiptr>;
     Buffer::bind(Buffer::Target::Uniform, 7, {
-        std::make_tuple(&buffer, 256, 13), {},
-        std::make_tuple(&buffer, 768, 64)});
+        BufferOffset(&buffer, 256, 13), {},
+        BufferOffset(&buffer, 768, 64)});
```

Comment:
Hmm whoops, no actually you were right, but only once I removed the <tuple> include, the expression is not ambiguous anymore! So, done

Reply:
Damn. Now the CI is failing on this file, because probably depending on the compiler `<tuple>` still gets included through an STL include chain

Reply:
Okay, that's not good. I'll try to find a better solution, having to specify the whole type here would be a usability regression.

Reply:
Yeah, the thing is that if people used the "implicit" idiom with just breace initializers, we wouldn't even need to provide the deprecated alternative as it would still work with the new API. The problem is if people used `make_tuple` explicitly, then it breaks without the deprecated signature. But if we keep the deprecated signature, the implicit idiom is broken as soon as `<tuple>` is included anywhere.

I don't really see any option here, it's kind of "pick your poison" moment. Either break *some* code, or have an ambiguous signatures that force spelling out the whole type.

Another non solution is, we just drop the triple change from this PR. The change that we do need, on our side, is actually just the `ArrayView` one.

Reply:
Right, good point.

If have to pick between porting halfway (and suffering the `<tuple>` include further) or breaking some (assuming rather rare) code, then I pick the breakage. So, can you drop the deprecated tuple overload altogether? And removing all deprecated (forward) tuple includes now as well.

Sorry for the extra work this caused 😅